### ISSUE TITLE
iTunes files disappearing from iTunes d/b, files themselves unaffected

### ISSUE THREAD

**Issue by willson556** [GetiPlayerAutomator/get-iplayer-automator#104]
_From [dynamite...@gmail.com](https://code.google.com/u/104100508614117650320/) on September 20, 2012 16:46:03_

What steps will reproduce the problem? 1.renaming get-iplayer-automator to "iPlayer permanent downloads"(to make name more pertinent for the oldies)
2.downloading iplayer file
3.playing file in iTunes What is the expected output? What do you see instead? to keep the database intact within iTunes
once a file (any TV show file) has been played, the file in iTunes d/b disappears, but file still in original folder What version of the product are you using? On what operating system? get iplayer automator v1.4.2
OS Mountain Lion, mac Please provide any additional information below. I renamed the get iplayer automator, as I have installed it on my parents' MacBook Air running 10.8;1 as the name is not intuitive for them.

Files are downloading fine, and are being added to iTunes as they should.

However, since the name change, every time I play ANY TV show (not just a TV show downloaded through the automator, but ANY TV show), the file mysteriously disappears from the iTunes d/b.

The original file remains in it's folder.

This is a problem with the automator, as all other files are unaffected - songs and movies do not self delete.

In other words, my poor parents cannot understand why the downloads - seemingly permanent - erase themselves, as they do after the 30 day limit in the iPlayer app itself.

It seems as if a simple name change has corrupted some iTunes plist (or equivalent) file. 

Re-installing iTunes does not solve this problem, and my parents have no idea how to rebuild these corrupt iTunes d/b files.

Please help!

_Original issue: http://code.google.com/p/get-iplayer-automator/issues/detail?id=104_
